src/coreclr/jit/emitxarch.cpp

```diff
@@ -1164,6 +1165,15 @@ emitter::code_t emitter::AddEvexPrefix(instruction ins, code_t code, emitAttr at
     return code;
 }
 
+emitter::code_t emitter::AddEvexbBitIfNeeded(const instrDesc* id, code_t code)
+{
+    if (hasEvexPrefix(code) && id->idIsEvexbContext())
```

Comment:
1. Do we already know that this is EVEX? Can we `assert(hasEvexPrefix(code))`?
2. Please add function header comment
3. Please fix type "perfix" above.